src/main/kotlin/com/plus/taxiapp/infra/store/member/MemberRepositoryImpl.kt

```diff
@@ -0,0 +1,26 @@
+package com.plus.taxiapp.infra.store.member
+
+import com.plus.taxiapp.domain.Member
+import com.plus.taxiapp.domain.MemberRepository
+import com.plus.taxiapp.domain.PaymentType
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Repository
+
+@Repository
+class MemberRepositoryImpl(
+    private val memberJpaRepository: MemberJpaRepository,
+): MemberRepository {
+    override fun updateDefaultPayment(memberId: Long, type: PaymentType, paymentId: Long): Member {
```

Comment:
🚫 **[ktlint]** <standard:function-signature> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should start on a newline